packages/analytics/dashboard-renderer/src/components/GoldenSignalsRenderer.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <MetricsProviderInternal v-bind="options">
+    <MetricsConsumer />
+  </MetricsProviderInternal>
+</template>
+<script setup lang="ts">
+import type { MetricCardOptions, RendererProps } from '../types'
+import { MetricsProviderInternal, MetricsConsumer } from '@kong-ui-public/analytics-metric-provider'
+import { computed } from 'vue'
+
+// Unlike AnalyticsChart, the metric card package doesn't currently expose its options
+// in a convenient interface.
+type ProviderProps = InstanceType<typeof MetricsProviderInternal>['$props']
+
+const props = defineProps<RendererProps<MetricCardOptions>>()
+
+const options = computed<ProviderProps>(() => ({
+  longCardTitles: props.chartOptions.longCardTitles,
+  description: props.chartOptions.description,
+  hasTrendAccess: true,
+  refreshInterval: 0,
+}))
+</script>
```

Comment:
We're not going to provide the flexibility yet for the host app to determine/define the metric cards they want to see, just keeping it at the three golden signals for now?

Reply:
That's correct.  A) There isn't any demand for that yet in any of the potential dashboards we want to create; and B) there's some complexity around which queries get issued, so it's not a feature we can just get for free.